package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "pure-brand-kit",
+  "version": "1.0.0",
+  "description": "A place to store global brand information for use across various Pure digital properties (Sass colour variables, typography etc)",
+  "main": "index.js",
+  "repository": "git@github.com:AgePartnership/pure-brand-kit.git",
+  "author": "Sam Hardacre <me@nocturnalmonkey.com>",
```

Comment:
This should *probably* be AP from an Intellectual Property perspective

Reply:
Will check with DO, this is how the others are set up. The repo is owned by AP so that probably covers IP

Reply:
Spoken to DO and he says it should probably should be sorted at some point but isn't a show-stopper

Reply:
Also wasn't sure about the license, checked with WO and he wasn't sure either so that's probably another one to check with DO

Reply:
That's also been raised and DO's reply covers the license for the time being.